reference/checkouts.v3.yml

```diff
@@ -826,7 +818,7 @@ definitions:
                       type: number
                       format: double
                     list_price:
-                      description: Item's list price, as quoted by the manufacturer/distributor.
+                      description: 'Item''s list price, as quoted by the manufacturer/distributor.'
```

Comment:
Please remove three single-quote/apostrophe characters. See main comment for details.